articles/aks/use-managed-identity.md

```diff
@@ -54,6 +54,9 @@ A successful cluster creation using managed identities contains this service pri
   }
 ```
 
+> [!NOTE]
+> For creating and using your own VNet, static IP address, or attached Azure disk where the resources are outside of the MC_* resource group, use the PrincipalID of the cluster to perform a role assignment. For more information on role assignment, see [Delegate access to other Azure resources](kubernetes-service-principal.md#delegate-access-to-other-azure-resources).
```

Comment:
While you are at it, can you also clarify that the principalID is that of the System Assigned Managed Identity?
"use the PrincipalID of the cluster"